### PR TITLE
Add a task to run manual tcb commands

### PR DESCRIPTION
**Pull request by drewmoseley**
This adds a task to launch a terminal with tcb-env-setup.sh already sourced and the DOCKER_HOST variable cleared.

### CODE REVIEW

**Comment by drewmoseley** [comment]
OK, I had to add a custom bashrc file to properly source tcb-env-setup and make sure that the alias was available.  This worked in my checked out sources.  I'll try to modify my config to pull a fresh project from my fork to test the full workflow.

**Comment by drewmoseley** [comment]
I was able to test this from scratch creating a new project using my fork of the templates.  From my POV this is good to go.

**Comment by andreriesco** [comment]
Please add it also to [update.json](https://github.com/torizon/vscode-torizon-templates/blob/dev/tcb/.conf/update.json), so it gets updated with the `try-template-update task`

**Comment by drewmoseley** [comment]
> Please add it also to [update.json](https://github.com/torizon/vscode-torizon-templates/blob/dev/tcb/.conf/update.json), so it gets updated with the `try-template-update task`

@andreriesco I don't follow.  What do you want me to list in update.json?

**Comment by andreriesco** [comment]
Actually, this script is to be used exclusively with the TCB IDE template, or do you think that being able to call this task can be beneficial on any other template also?

**Comment by drewmoseley** [comment]
I was only thinking of the TCB template.  I don't currently have any need for it in the other templates.

**Comment by andreriesco** [comment]
I think it can be added to the `.conf` of the tcb template then, [here](https://github.com/torizon/vscode-torizon-templates/tree/dev/tcb/.conf), the `scripts` folder is generally for scripts that have uses in multiple templates. But inside the `tcb` folder it's easier to maintain it and to update it on the project

**Comment by drewmoseley** [comment]
@andreriesco I still don't understand.  What is it I should add to `./tcb/.conf/update.json`?  I don't understand how that file is used.

**Comment by andreriesco** [comment]
This file is for the `try-update-templates` task to know which files it should update (copy and open the merge window) from the templates repository into the project. Files which are specific of the template. So here for example it updates this three files:
```
[
    {
        "source": ".github/workflows/build-torizoncore.yaml",
        "target": ".github/workflows/build-torizoncore.yaml"
    },
    {
        "source": ".gitlab-ci.yml",
        "target": ".gitlab-ci.yml"
    },
    {
        "source": "tcbuild.yaml",
        "target": "tcbuild.yaml"
    }
]
```
And with this script at `tcb/.conf`, it will become:
```
[
    {
        "source": ".github/workflows/build-torizoncore.yaml",
        "target": ".github/workflows/build-torizoncore.yaml"
    },
    {
        "source": ".gitlab-ci.yml",
        "target": ".gitlab-ci.yml"
    },
    {
        "source": "tcbuild.yaml",
        "target": "tcbuild.yaml"
    },
    {
        "source": ".conf/bashrc.tcb",
        "target": ".conf/bashrc.tcb"
    },
]
```

**Comment by drewmoseley** [comment]
OK.  Done.

**Comment by andreriesco** [comment]
Also, the `bashrc.tcb` needs to be moved from `scripts/bash` to [tcb/.conf](https://github.com/torizon/vscode-torizon-templates/tree/dev/tcb/.conf)

File: scripts/createFromTemplate.ps1 [hunk 1]
```diff
@@ -141,6 +141,7 @@ Copy-Item "$templateFolder/../scripts/createDockerComposeProduction.ps1" "$locat
 Copy-Item "$templateFolder/../scripts/torizonPackages.ps1" "$location/.conf"
 Copy-Item "$templateFolder/../scripts/tasks.ps1" "$location/.vscode"
 Copy-Item "$templateFolder/../scripts/bash/tcb-env-setup.sh" "$location/.conf"
+Copy-Item "$templateFolder/../scripts/bash/bashrc.tcb" "$location/.conf/.bashrc.tcb"
```
Comment (andreriesco) [inline scripts/createFromTemplate.ps1]: Also, this can be removed as the script will be inside the `tcb` folder

Comment (drewmoseley) [reply scripts/createFromTemplate.ps1]: Done and tested.

**Comment by drewmoseley** [comment]
@andreriesco anything else here?

**Comment by andreriesco** [comment]
No, now it looks good to me. I will merge it, thanks

**Pull request merged**